pkgs/development/libraries/kde-frameworks/default.nix

```diff
@@ -28,7 +28,7 @@ existing packages here and modify it as necessary.
 
 let
 
-  minQtVersion = "5.14";
```

Comment:
It seems `kimageformats` framework has a patch included which fails to apply when rebasing this on top of master because it was already included in `5.83`. Maybe they transformed the warning in an error?

Reply:
Yeah 0c556f31caba6f198b2d77f1e4b33d53dd58ee81 needs to be reverted, the patch would technically still apply, but in reverse (and there is a comment on that commit mentioning that the change is in 5.83). The patch applying only if reversed causes nix to panic and bail out.